ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -54,7 +54,9 @@ function set_common_variables () {
 function check_idp () {
     # Check if runtime_env exists and then check if $USERS is not empty
     if [ -f "${SHARED_DIR}/runtime_env" ]; then
+        set +x
         source "${SHARED_DIR}/runtime_env"
+        set -x
```

Comment:
Shall we remove those debug code before merging the PR ?

Reply:
@xingxingxia Should we remove these completely from script? What do you suggest.

Reply:
If you mean the one below: we'd like to keep [it](https://github.com/openshift/release/blob/80a6e88d9f26e004255d94161c49b56c9063dbdb/ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh#L3-L6) which is useful when checking failed logs more quickly. `grep -nr 'set -[^ ]*x' ci-operator/config` shows many others used too.
https://github.com/openshift/release/blob/80a6e88d9f26e004255d94161c49b56c9063dbdb/ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh#L3-L6 .